### ISSUE TITLE
Localization progress and contribution methods

### ISSUE THREAD

**Issue by dyphire** [mpvnet-player/mpv.net#576]
I noticed that mpv.net started adding localized extension support after migrating to Net.6, but it seems to be still in the initialization stage.
Open a question here to focus on localization progress and subsequent contribution methods.

**Comment by stax76** [comment]
I agree now that it's important, and I really want to do it, but it's certainly difficult to do, because of that I don't know when is the right time to start.

**Comment by zhongfly** [comment]
Mainly the configuration editor and input editor need to be localized.

**Comment by zhongfly** [comment]
Using transifex don't look like a good idea because it isn't free or cheap.

**Comment by stax76** [comment]
Yes, it's free for open source projects. It's apparently used for the translation of mkvtoolnix, I found it by looking at how mkvtoolnix is translated.

I first translate the menu and bindings and learn how transifex works, it seems to be complex and slow and feature rich. After translating the menu and bindings and learning how things work, I invite translators to translate the menu and bindings.

I want to rewrite/improve the conf editor, it's probably better not to translate it before doing the rewrite, because of that the conf editor isn't yet integrated in the translation powershell script. Both the rewrite and translation of the conf editor will be difficult, so I don't have a schedule for it.

**Comment by stax76** [comment]
@dyphire I saw your request on transifex, I'll need a couple of days until everything is ready, my main concern is data loss, which is more likely in the case of working with a new and unfamiliar system.

**Comment by dyphire** [comment]
> @dyphire I saw your request on transifex, I'll need a couple of days until everything is ready, my main concern is data loss, which is more likely in the case of working with a new and unfamiliar system.

OK, I will wait for it.

**Comment by stax76** [comment]
You should now be able to translate the menu. There are 145 words or sentences. I hope the AI translation works well in Chinese, it worked well in German.

It took me a while to understand everything and put it together. I use now the transifex CLI client to push and pull the data.

**Comment by dyphire** [comment]
> You should now be able to translate the menu. There are 145 words or sentences. I hope the AI translation works well in Chinese, it worked well in German.

The Chinese (zh_CN) translation has been completed. The results of AI translation are good, with only a small amount requiring correction and embellishment.
